docs/UserGuide.md

```diff
@@ -269,10 +269,10 @@ Example:`--view` or `-v`
   Example:`--timezone UTC+08` or `-t UTC-1030`
 
 Here's an example of a command using all parameters:<br>
-`java -jar RepoSense.jar --repo https://github.com/reposense/RepoSense.git --output ./report_folder --since 21/10/2017 --until 21/11/2017 --formats java adoc js --view --ignore-standalone-config --timezone UTC+08`
+`java -jar RepoSense.jar --repo https://github.com/se-edu/addressbook-level4.git --output ./report_folder --since 1/1/2017 --until 1/1/2018 --formats java adoc xml --view --ignore-standalone-config --timezone UTC+08`
```

Comment:
why this change?

Reply:
The previous example leads to an empty report as no commit infos were available during that period given the parameters. #756

Reply:
I see. That is fine but avoid using a repository from different organization in our documentation unless its for acknowledgement. Also, while the date range is fine, try to use dates that allows user to easily tells apart the day and the month. E.g. `31/10`.

Reply:
Ok. We use back RepoSense in this case?

Reply:
Yes

Reply:
> That is fine but avoid using a repository from different organization in our documentation unless its for acknowledgement.

I suggested using se-edu because using `RepoSense` multiple times in the command can be confusing to the reader as it is also the name of the tool.

Reply:
But I'm OK to keep using RepoSense repo too.